source/views/guides/getting-started.html.slim

```diff
@@ -50,10 +50,10 @@ main
             pre.demo__example-snippet
               code
                 | [dependencies]
-                  diesel = "0.6.1"
-                  diesel_codegen = { version = "0.6.1", default-features = false, features = ["nightly", "postgres"] }
-                  dotenv = "0.8.0"
-                  dotenv_macros = "0.8.0"
+                  diesel = "0.6"
```

Comment:
I don't think this is the same, is it? `diesel = "0.6"` is the same as `diesel = "0.*"` as far as I'm aware.